packages/x-data-grid-pro/src/tests/columns.DataGridPro.test.tsx

```diff
@@ -165,12 +165,13 @@ describe('<DataGridPro /> - Columns', () => {
       fireEvent.doubleClick(separator);
       await microtasks();
       expect(onColumnWidthChange.callCount).to.be.at.least(2);
-      const widthArgs = onColumnWidthChange.args.map((arg) => arg[0].width);
-      const isWidth114Present = widthArgs.some((width) => width === 114);
-      expect(isWidth114Present).to.equal(true);
-      const colDefWidthArgs = onColumnWidthChange.args.map((arg) => arg[0].colDef.width);
-      const isColDefWidth114Present = colDefWidthArgs.some((width) => width === 114);
-      expect(isColDefWidth114Present).to.equal(true);
+      const widthArgs = onColumnWidthChange.args.map((arg) => Math.round(arg[0].width));
+      expect(widthArgs).to.deep.equal([120, 64]);
```

Comment:
IMHO, this needs a review from xGrid as it looks like something functional has changed. 🤔

Reply:
Yeah, the scrollbar is not rendered, so the size is different. 

It took me a while to find the actual issue.

As I said, there are two options:

1. Test all without scrollbar
2. Enable scrollbars to all playwright tests if the scrollbar is important

Reply:
Noticed the CI is still erroring on the changes, I might have to go with 2 regardless for this specific error 😓

Reply:
> As I said, there are two options:
> 
> 1. Test all without scrollbar
> 2. Enable scrollbars to all playwright tests if the scrollbar is important

But why did the behavior of the scrollbar change if the `playwright` package had not been bumped? 🤔 
Is it the same issue blocking us here: https://github.com/mui/mui-x/pull/13672?

Reply:
No, they are unrelated.

Behaviour is based on the browser arguments, karma is using the playwright browser executable only, and passing its own arguments in, which means it shows scrollbars in headless mode.

Playwright on the other hand, passes their own arguments in, which means it will hide the scrollbars in headless mode .

Reply:
I've reverted the changes specific to scrollbar, as the 2nd option seems to fix it cleanly.

Reply:
Cool. 👍
LGTM then, I've restarted a flaky unit test. 😉